### ISSUE TITLE
Dotmim.Sync.SyncException : 'SQLite Error 20: 'datatype mismatch'.'

### ISSUE THREAD

**Issue by oxedgar** [Mimetis/Dotmim.Sync#879]
Hello Sébastien and DMS Team,

I get this error wehn i try to make this:
```
            richTextBox1.Text = "Nouvelle synchro forcée vers le serveur en cours";
            var syncResult = await agent.SynchronizeAsync(SyncType.ReinitializeWithUpload);
            richTextBox1.Text = syncResult.ToString();
```

I Cannot found Where is my problem. can you help me?

For info, normal synchro is working. This appears wehn i try to force send client db to server.
In my case, the sqlite db is a source of datas. for my application, i need to use DMS to sync all clients with server but only one client is "masater" and this one is only able to create rows on database and other clients is only used to update sqlite database.

Thank for you good job

**Comment by Mimetis** [comment]
Seems you have a mismatch on column type
Did you alter the schema ?

**Comment by oxedgar** [comment]
in fact, I would have liked to import my SQLITE database into my test SQL. However, I failed to do so. So I created my tables manually and then ran a test sync on my first test client, which went well. then I wanted to import another client but I get this error.

I don't mind importing content from my second client because the server data is dummy. How can I force the creation of the full DB in SQL using a SQLITE as the source? This could solve my problem because the types are managed by DMS! the software that I am looking to synchronize is currently only compatible with SQLITE3. I speak French if it's easier because my English is limited!
